crates/astria-composer/src/searcher/collector.rs

```diff
@@ -10,6 +10,7 @@ use ethers::providers::{
     Ws,
 };
 use humantime::format_duration;
+use proto::native::sequencer::v1alpha1::ChainId;
```

Comment:
I'm wondering if we should rename `proto` to `types` or something, and turns the protos into just a module within that crate. it's a bit confusing to have the native types inside a crate called `proto`

Reply:
Yeah, definitely. The naming is not optimal. We should rename `astria-proto` and make `proto` a submodule of that. I don't know what to be honest. I am leaning toward `astria-api` or `astria-rpc` or `astria-public`. Or even just `astria`. Not in this PR obviously, but agree.